source/branding.html.erb

```diff
@@ -21,27 +24,37 @@ title: Brand Guidelines
       layout. Additionally, the circular logo and the
       name must always appear together.
     </p>
+    <p style="font-size: .7rem;">
+      * In rare occasions the horizontal logo is needed.
+      For these instances please contact <%= link_to "marketing@#{chef_domain}", "mailto:marketing@#{chef_domain}" %> for a request.
+    </p>
   </div>
   <div class="small-12 large-6 columns">
-    <div class="row" style="text-align:center;">
+    <div class="row example" style="text-align:center;">
```

Comment:
The example class was stolen from another page. I think we used them in the code snippet examples. So dont want to mess with that. As for the text alignment, I just followed suit on how it was originally done. yeah, it could probably be done better and Foundation probably has a center tag as well.